packages/core/test/tracing/reactnavigation.test.ts

```diff
@@ -210,6 +210,41 @@ describe('ReactNavigationInstrumentation', () => {
     );
   });
 
+  test('navigation action not ', async () => {
```

Comment:
m: This test case name and implementation looks incomplete. Should this test a different scenario than the above?

Reply:
Thank you, no just a bad copy paste.